javasphinx/formatter.py

```diff
@@ -115,6 +115,9 @@ def output_type_params(type_params, output):
 
 @formatter
 def output_declaration(declaration, output):
+    if declaration.annotations is None:
```

Comment:
I think this change could be reverted if the suggested changes are made in `compiler.py`.